src/rollup/index.ts

```diff
@@ -204,11 +219,9 @@ export default function rollup(
 				const singleInput =
 					typeof inputOptions.input === 'string' ||
 					(inputOptions.input instanceof Array && inputOptions.input.length === 1);
-				//let imports: string[], exports: string[];
 				if (!inputOptions.experimentalPreserveModules) {
 					if (singleInput) {
 						for (const chunk of chunks) {
-							if (chunk.entryModule === undefined) continue;
```

Comment:
Should this not be replaced with `chunk.entryModules.length === 0` or similar to retain the logic?

Reply:
I am not sure the old logic was correct. Now we no longer have a single chunk as soon as there is more than one chunk no matter for what reason another chunk was created.